### ISSUE TITLE
make GCTools accessible to screen readers

### ISSUE THREAD

**Issue by j-paquette** [wet-boew/cdts-sgdc#1496]
This was identified after a11y testing of cdts (https://ado.intra.dmz/projectcollection/EWS-SWE/_queries/edit/1045466/?triage=true):

a11y test report (134 - Using script to remove focus when focus is received):
- when opening the modal GCTools, the focus gets removed and instead focuses the whole of the screen rather then the modal itself, or modal title. 
- when pressing down on the arrow key, SR will read a skip link.

a11y test report (322 - Page does not work in mobile view):
- menu modal is incorrect, allows for background scrolling
- screen reader cannot read "Collapse" within the GC-Tools dropdown due to incorrect Modal pattern
- using left and right arrow keys is not typical for Menus that drop-down and not to the side
- keyboard and screen reader users will not be able to properly navigate modal controls

a11y test report (352 - Aria markup present but without expected behaviour or design patterns):
- within mobile view, (400% Reflow) the icon label "Menus" label and image visually presents to the user that when activated, a list would drop down. 
- the user is presented with a modal instead.
- cognitive and screen reader users expect a drop-down Menu list when activating the icon, but get a modal instead

Suggestion: change GCTools from a modal to a menu bar?
